fms/datasets/arrow.py

```diff
@@ -55,7 +57,8 @@ def __init__(
         rank: int = 0,
         world_size: int = 1,
         column_name: str = "tokens",
-        max_seq_len: int = 4096,
+        max_seq_len: Optional[int] = None,
```

Comment:
what happens when max_seq_len=None?

Reply:
assume that the whole sequence is fixed length rather than trying to chunk it. i.e. we've fully pre-processed the data into batches of the correct length.

it might even be better to just get rid of this logic all together. it only really makes sense if the length of the docs is greater than max seq len of the model, but if it is then it's harder to get the state dict characteristics we also want.